src/test/java/com/google/sps/data/friend_map/FriendsMapCreationTest.java

```diff
@@ -48,85 +45,75 @@
   private UserNode userC;
```

Comment:
These should not be here. You want to declare in each test case.

Instead of `userA = new UserNode(...);`, you'd want `UserNode userA = new UserNode(...);`

Reply:
Ok, got it! I took these UserNode declarations out and moved them to be declared within the test cases.